notebook2/notebook/templates/notebook.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en" {{ 'class=dark' if session['dark'] }}>
+  <head>
+    <title>Hail Notebooks</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"/>
+    <link rel="stylesheet" type="text/css" href="/static/css/main.css">
+    <link rel="stylesheet" type="text/css" href="/static/css/components/header.css">
+    <link rel="stylesheet" type="text/css" href="/static/css/pages/notebook.css">
+  </head>
+  <body id='theme-site'>
+    {% include 'header.html' %}
+    <div id="main">
+      <div id="notebook" class="centered">
+        <form action="{{ form_action_url }}" method="post">
+          <p>
+            <label for="image">Image:</label>
+            <select name="image" id="image">
+              {% for image in images %}
+              <option value="{{ image }}"{{'selected' if image == default else ''}}>{{ image }}</option>
+              {% endfor %}
+            </select>
+          </p>
+          <input type="submit" value="Start a Hail Notebook" />
```

Comment:
I think we should just use a `button` tag. AFAICT, a `button` tag (under the description of the type attribute):
> [...] submits the form data to the server. This is the default [...]

[MDN Button Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button)

Reply:
input type=html is the canonical way afaik for html forms: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit 

"
input type="submit" buttons are used to submit forms. If you want to create a custom button and then customize the behavior using JavaScript, you need to use input type="button", or better still, a button element.
"

Reply:
Additional comment on this: use of `input type='submit'` predates `button`. I remember using the `button` for styling reasons...it really doesn't matter to me which one we use.

Notebook 1 used type='submit', so I kept it. If you'd like me to change that in this PR, no problem!